### ISSUE TITLE
Install from source of origin-ci-tool does not work on Mac venv

### ISSUE THREAD

**Issue by smarterclayton** [openshift/origin-ci-tool#36]
```
pip install git+https://github.com/openshift/origin-ci-tool.git --follow-dependency-links
```

inside of a normal mac venv with 

```
virtualenv venv --system-site-packages
```

does not work:

```
○ pip install git+https://github.com/openshift/origin-ci-tool.git --follow-dependency-links

Usage:
  pip install [options] <requirement specifier> ...
  pip install [options] -r <requirements file> ...
  pip install [options] [-e] <vcs project url> ...
  pip install [options] [-e] <local project path> ...
  pip install [options] <archive url/path> ...

no such option: --follow-dependency-links
```

That's with the latest pip.

Removing that, I get:

```
(.venv)○ pip install git+https://github.com/openshift/origin-ci-tool.git
Downloading/unpacking git+https://github.com/openshift/origin-ci-tool.git
  Cloning https://github.com/openshift/origin-ci-tool.git to /var/folders/zw/j6656mbn1rv4fnqqdggss8h80000gn/T/pip-DUBSe9-build
  Running setup.py (path:/var/folders/zw/j6656mbn1rv4fnqqdggss8h80000gn/T/pip-DUBSe9-build/setup.py) egg_info for package from git+https://github.com/openshift/origin-ci-tool.git

    warning: no files found matching 'vagrant/Vagrantfile'
Downloading/unpacking Click (from oct==0.1.0)
  Downloading click-6.6-py2.py3-none-any.whl (71kB): 71kB downloaded
Downloading/unpacking ansible==3.0.0 (from oct==0.1.0)
  Could not find a version that satisfies the requirement ansible==3.0.0 (from oct==0.1.0) (from versions: 1.7.1, 2.1.3.0, 1.6.10, 1.5.2, 1.4.3, 1.4.2, 1.4.4, 1.1, 1.2.2, 1.5.1, 1.6.5, 1.4.5, 2.1.0.0, 2.1.1.0, 1.5.5, 1.7, 1.5, 1.5.4, 1.6.4, 1.6.1, 1.6.2, 2.0.0.2, 1.8.2, 2.2.0.0, 1.6.9, 1.9.3, 2.0.0.1, 1.6.7, 1.8.4, 1.9.4, 1.7.2, 1.2.1, 2.1.2.0, 1.8.3, 1.6.3, 2.0.0.0, 1.6.8, 1.2.3, 1.5.3, 1.3.2, 1.8.1, 1.3.0, 1.9.2, 1.4.1, 1.9.5, 2.0.2.0, 1.0, 1.6, 1.9.6, 1.9.0.1, 1.9.1, 1.4, 1.3.1, 1.3.4, 1.6.6, 2.0.1.0, 1.2, 1.8, 1.3.3)
Cleaning up...
No distributions matching the version for ansible==3.0.0 (from oct==0.1.0)
Storing debug log for failure in /Users/clayton/.pip/pip.log
```

**Comment by smarterclayton** [comment]
```
$ pip --version
pip 9.0.1 from /usr/local/lib/python2.7/site-packages (python 2.7)
$ virtualenv .venv --system-site-packages
New python executable in .venv/bin/python
Installing setuptools, pip...done.
$ pip version
$ source .venv/bin/activate
(.venv)$ pip --version
pip 1.5.6 from /Volumes/development/projects/origin/src/github.com/openshift/origin/.venv/lib/python2.7/site-packages (python 2.7)
(.venv)
```

**Comment by stevekuznetsov** [comment]
This should be fixed with the MANIFEST update in #38 and using `--process-dependency-links` instead of `--follow-dependency-links`.
